### ISSUE TITLE
Cards order are not saved

### ISSUE THREAD

**Issue by gusarg81** [kiswa/TaskBoard#520]
**Describe the bug**
When I move cards to change orders (in the same column), that change are not saved after closing and open again TaskBoard

**To Reproduce**
1. Create some cards in the same column
2. Drag and drop to change order
3. Close TaskBoard site. Open again and you will see that orders was not changed.

**Please complete the following information:**
- OS: Ubuntu 18.04 server
- TaskBoard Version: latest 1.0.1

**Comment by kiswa** [comment]
I'll look into this. Thanks for the report.

**Comment by gusarg81** [comment]
What I saw doing more testing, since now I use now TaskBoard in replace of Kanboard for daily basis, that does not always happen; sometimes does save the position, sometimes does not.

**Comment by gusarg81** [comment]
I have more for you: sometimes when I create a new Task (or modify one), others task in the same column (I didn't checked the other columns), change their order (like random). For example;

- I've modified the first task in the column. After saved, the next task in this column, just placed in a random order more bellow others tasks.

Have in mind that I am using mysql as backend, in case is related.

**Comment by 2Belette** [comment]
@kiswa unfortunately I can tell you that this issue still occurs on 1.0.2..
I realized that it is even worst : editing a task is changing the others tasks order on the same column like @gusarg81 described but even worst : the tasks from other columns are moved back to the first column :(

**Comment by kiswa** [comment]
WTH?! I'll add it to the list...

**Comment by 2Belette** [comment]
Yep, sorry for that...I have started to create a quite big board for a professional project and I realized that every messed-up when I edited a card. I have to put everything on old as having 70 cards re-ordered randomly and get back to the original columns it have been created is unusable (+ the kick-out issue it is almost not possible to use it) :( If I can help you troubleshoot let me know

**Comment by kiswa** [comment]
I feel like something went very wrong with this update, but I don't know how it happened. I tested column order saving, and the forced logout problem. I apologize for not having a better testing setup to find these issues _before_ they get released.

I'll create a test DB with at least 100 tasks on it so I can more easily see any bugs with large amounts of tasks.

**Comment by 2Belette** [comment]
👍 I would say that this first 1.0 version is almost usable now there is a workaround for kickoff issue #514, it would only need an urgent fix for this Cards order not saved as this is all the kanban purppose and it is not usable without this :(

**Comment by 2Belette** [comment]
@kiswa I hope this would give you an insights on where to look for : I realized that the moving a card using drag & drop is the one which is not keeping the card on the destination column BUT if I edit the card and select the column in the drop down menu, the card is kept on the right column even after refresh/logout.

btw & of topic : you mentioned you have added 100 cards in your testing setup, are you experimenting as well slownest on your browser side when you edit/save a card ? (for me it take about 15-20sec for every action with 70 cards)

**Comment by reivilibre** [comment]
I seem to get about 1 chance to move a ticket before no more moves will work on that page load (I have to refresh after every move). Odd.
